src/media/css/detail/reviews.styl

```diff
@@ -57,3 +57,11 @@
         margin-bottom: -60px;
     }
 }
+
+.text-small {
+	font-size: 12px;
+    }
+
+hr {
```

Comment:
Not a fan of this in case we add other `<hr>`s in the future, something a bit more specific would be great - does it look the same in other browsers? These used to need a height/padding check.